assets/scss/templates/_guidelines.scss

```diff
@@ -141,51 +142,75 @@
 }
 
 .buttons-card {
-  height: 235px;
   border-radius: 5px;
-  overflow: hidden;
   margin-bottom: 30px;
-  width: 300px;
+  width: 25%;
+  flex-direction: column;
+  display: flex;
+  align-items: center;
   text-align: left;
+
+  &:before {
+    opacity: 0;
+    border-radius: 3px;
```

Comment:
it's not the default radius?

Reply:
Nop, create now! thanks @fcoronelmakingsense !